cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java

```diff
@@ -321,6 +321,7 @@ public void testTemplate() {
         assertThat(dep.getSpec().getTemplate().getSpec().getSchedulerName(), is("my-scheduler"));
         assertThat(dep.getSpec().getTemplate().getSpec().getTopologySpreadConstraints(), containsInAnyOrder(tsc1, tsc2));
         assertThat(dep.getSpec().getTemplate().getSpec().getEnableServiceLinks(), is(false));
+        assertThat(dep.getSpec().getTemplate().getSpec().getTolerations(), is(singletonList(toleration)));
```

Comment:
I think this is wrong and is causing the test failure?

Reply:
Thanks for pointing this out

The line mentioned is from the main branch. I noticed that there is another line using the variable assertToleration which was missing, and therefore test failure. I've added the assertToleration variable now, and the test should pass

Reply:
I don't think you handled it well - or maybe I misled you. If you compare it with the main branch, this is the correct line: https://github.com/strimzi/strimzi-kafka-operator/blob/ff128d8fbf9e964671c6b6367e571c69fa172ce6/cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java#L297 ... you are restoring the removed behavior, so it will likely fail when running the tests. (also, please just delete the wrong line, not comment it out)

Reply:
Sorry, my bad. I got confused by your comment and the error message in the pipeline. I will correct it to the right line, the one you are referring to, and remove the old behavior.

Thanks for your patience :-)